### ISSUE TITLE
xero-api with qbo_api gem exceptions handling

### ISSUE THREAD

**Issue by blahutka** [minimul/xero-api#1]
Hi Christian, thank you for this gem.

I discovered problem with xero-api gem. when I use it together with `qbo_api` in the exact order first Xero second QBO
```
# Gemfile 
gem 'xero-api'
gem 'qbo_api', "~> 1.8.4"
```

It doesn't handle unauthorized exceptions and raises parsing error
```
JSON::ParserError at /agendas/1/supplier_automation/download_all
784: unexpected token at 'oauth_problem=token_expired&oauth_problem_advice=The%20access%20token%20has%20expired'
```
----
When I switch the loading order it seems ok
```
# Gemfile 
gem 'qbo_api', "~> 1.8.4"
gem 'xero-api'
```
```
Xero::Api::Unauthorized at /agendas/1/supplier_automation/download_all
oauth_problem=token_expired&oauth_problem_advice=The%20access%20token%20has%20expired`
```

**Comment by blahutka** [comment]
It tries to load qbo_api exceptions instead of xero-api
![image](https://user-images.githubusercontent.com/480/64771532-709d4b00-d54f-11e9-97b9-dd99cd109ad3.png)

**Comment by minimul** [comment]
Thanks @blahutka  for finding and reporting this. I believe the problem is improper name spacing. See https://github.com/minimul/xero-api/tree/raise-namespace but I didn't try to reproduce. Can you give it a go? Thanks much.

**Comment by blahutka** [comment]
@minimul Yes, this is working. Thank you

**Comment by minimul** [comment]
Fixed and released in v 1.0.1
